14198: Incorporated Sun's feedback in the 'Deploying Che on AWS' sect… (#781)
codenvy/che-docs: src/main/pages/che-7/overview/proc_preparing-the-aws-system-for-installing-che.adoc

}
----

> . Configure the four DNS nameservers on the `my-ide.cloud` DNS. Note that if you are using your own DNS provider, updating the record takes a few hours.

Reviewer: ```suggestion
. Configure the four DNS nameservers on the `my-ide.cloud` DNS. Note that when a custom DNS provider, updating the record takes a few hours.
```

$ aws iam create-service-linked-role --aws-service-name "elasticloadbalancing.amazonaws.com"
----

> . Add hosts on route 53 with this given hostname *link:https://console.aws.amazon.com/route53/home?region=eu-west-1#hosted-zones[]:*. Ensure that you include the colon (*:*) at the end of this URL.

Reviewer: ```suggestion
. Add hosts on route 53 with this given hostname ++https://console.aws.amazon.com/route53/home?region=eu-west-1#hosted-zones:++. Ensure that you include the colon (`:`) at the end of this URL.
```

Reviewer: @supriya-bharadwaj, you need to build and test the doc. The way you formatted it, no colon would be included in the URL (it would be behind it).

Author: @rkratky , I did click this link to check where it leads me to. But, it only took me to the login page for AWS. Since I do not have login credentials for it and we did not have enough time, I did not report this. Maybe I should have.
